aether-kernel/aether/kernel/api/serializers.py

```diff
@@ -281,23 +293,21 @@ class Meta:
 class EntityListSerializer(serializers.ListSerializer):
 
     def create(self, validated_data):
-        errors = []
         entities = []
         # remove helper field and validate entity
+        # only do this once since all schema decoratos are the same.
```

Comment:
Is this assumption true?
Can't I push entities from different projects?

Reply:
You're totally right.

Reply:
Could you add a test in which you send two entities from two different projects and see what's happening there?